manifests/job.pp

```diff
@@ -113,13 +114,13 @@
     name                => $name,
     jobtype             => $jobtype,
     fileset_real        => $fileset_real,
-    pool                => $pool,
+    pool                => $pool.lest || { $bacula::client::default_pool },
```

Comment:
Basically this allows `$pool` to be passed in, but if undef, returns the value of `$bacula::client::default_pool`.  Am I understanding that correctly?

Reply:
Yes!